assembly/src/tokens/mod.rs

```diff
@@ -226,7 +246,12 @@ impl<'a> Token<'a> {
         assert_eq!(Self::WHILE, self.parts[0], "not a while");
         match self.num_parts() {
             0 => unreachable!(),
-            1 => Err(ParsingError::missing_param(self)),
+            1 => Err(
+                    ParsingError::missing_param(
+                        self,
+                        "while <true>",
+                    )
```

Comment:
The correct format here would be `while.true`.

Reply:
Changed.